eval/adjust.rkt

```diff
@@ -379,19 +388,20 @@
      (list (- slack (minlog z)))]
 
     [(ival-pow2)
+     ; same as multiplication
      (define x (first srcs))
      (list (logspan x))]
     
     ; TODO
     [(ival-erfc ival-erf ival-lgamma ival-tgamma ival-asinh ival-logb)
      (list (get-slack))]
+    ; TODO
     [(ival-ceil ival-floor ival-rint ival-round ival-trunc)
```

Comment:
rounding functions need to have something like: if the distance is exactly 1 between two endpoints, then return slack as an `ampl`

Reply:
Yeah sure. I think these functions come up rarely but yeah we could do a better job, yeah, with the exactly-1-apart test.